front_end/panels/react_devtools_placeholder/react_devtools_placeholder-meta.ts

```diff
@@ -0,0 +1,47 @@
+// Copyright (c) Meta Platforms, Inc. and affiliates.
+// Copyright 2020 The Chromium Authors. All rights reserved.
+// Use of this source code is governed by a BSD-style license that can be
+// found in the LICENSE file.
+
+import * as i18n from '../../core/i18n/i18n.js';
+import * as Root from '../../core/root/root.js';
+import * as UI from '../../ui/legacy/legacy.js';
+
+import type * as ReactDevToolsPlaceholder from './react_devtools_placeholder.js';
+
+const UIStrings = {
+  /**
+   * @description Title of the React DevTools placeholder panel, plus an emoji symbolizing React Native
+   */
+  reactDevToolsWelcome: '⚛️ React DevTools',
+
+  /**
+   * @description Command for showing the React DevTools placeholder panel
+   */
+  showReactDevTools: 'Show React DevTools panel',
+};
+const str_ = i18n.i18n.registerUIStrings('panels/react_devtools_placeholder/react_devtools_placeholder-meta.ts', UIStrings);
+const i18nLazyString = i18n.i18n.getLazilyComputedLocalizedString.bind(undefined, str_);
+
+let loadedReactDevToolsPlaceholderModule: (typeof ReactDevToolsPlaceholder|undefined);
+
+async function loadReactDevToolsPlaceholderModule(): Promise<typeof ReactDevToolsPlaceholder> {
+  if (!loadedReactDevToolsPlaceholderModule) {
+    loadedReactDevToolsPlaceholderModule = await import('./react_devtools_placeholder.js');
+  }
+  return loadedReactDevToolsPlaceholderModule;
+}
+
+UI.ViewManager.registerViewExtension({
+  location: UI.ViewManager.ViewLocationValues.PANEL,
+  id: 'react-devtools-placeholder',
+  title: i18nLazyString(UIStrings.reactDevToolsWelcome),
+  commandPrompt: i18nLazyString(UIStrings.showReactDevTools),
+  order: 110,
+  persistence: UI.ViewManager.ViewPersistence.PERMANENT,
+  async loadView() {
+    const ReactDevToolsPlaceholder = await loadReactDevToolsPlaceholderModule();
+    return ReactDevToolsPlaceholder.ReactDevToolsPlaceholder.ReactDevToolsPlaceholderImpl.instance();
+  },
+  experiment: Root.Runtime.ExperimentName.REACT_DEVTOOLS_PLACEHOLDER_PANEL_ENABLE,
```

Comment:
> if flag is not enabled

I'd love to, but can't find any pattern to invert an experiment check in this way in the codebase 😐. What we will have instead is one configuration point where it should be clear we'll disable one experiment over the other.

```js
Root.Runtime.experiments.enableExperimentsByDefault([
  Root.Runtime.ExperimentName.REACT_NATIVE_SPECIFIC_UI,
  Root.Runtime.ExperimentName.REACT_DEVTOOLS_PLACEHOLDER_PANEL_ENABLE,
]);
```

Reply:
Bit surprised CDT doesn't have the convention of "disabled experiment". In that case, sounds like it would it be simpler to

1. remove the existing experiment field from RDT view registration
1. modify the existing RDT view registration to load this placeholder based on the existing experiment value.

Depending on where we check the experiment, we can mark the RDT experiment registration as "restart required" too.

That way we have a single deterministic experiment and always have the RDT panel visible (goal of this PR).

Reply:
We could do this, but I don't think the juice is worth the squeeze, as we'll remove this placeholder soon enough. Main blocker is the RDT panel hasn't landed yet.

Reply:
Discussed offline with @hoxyq — will _remove_ my experiment flag and enable by default. Ruslan will build on top of this to use this panel as the RDT panel.